docs/documentation/variables.rst

```diff
@@ -42,28 +45,89 @@ The other path elements depend of the variable. The number of path elements is n
 Serialization
 ***************
 
+File format
+***********
+
 For writing input and output files, FAST-OAD relies on the path in the variable names.
 
-For example, for the three variables above, the matching part in XML file will be:
+For example, for the three variables above, the matching XML file will be:
 
 .. code-block:: xml
 
-    <data>
-        <geometry>
-            <wing>
-                <area units="m**2">150.0</area>
-            </wing>
-        </geometry>
-        <weight>
-            <fuselage>
-                <mass units="kg">10000.0</mass>
-                <CG>
-                    <x units="m">20.0</x>
-                </CG>
-            </fuselage>
-        </weight>
-    </data>
-
-**Note**: *Units are given as a string according to* `OpenMDAO units definitions <http://openmdao.org/twodocs/versions/latest/features/units.html>`_
+    <FASTOAD_model>
+        <data>
+            <geometry>
+                <wing>
+                    <area units="m**2">150.0</area>
+                </wing>
+            </geometry>
+            <weight>
+                <fuselage>
+                    <mass units="kg">10000.0</mass>
+                    <CG>
+                        <x units="m">20.0</x>
+                    </CG>
+                </fuselage>
+            </weight>
+        </data>
+    </FASTOAD_model>
```

Comment:
Maybe it is obvious, but shouldn't we say that each : corresponds to an indent in the input and output files ?

Reply:
Well, that has nothing to do with indents (we talk about XML), but I see your point. Indeed, I though that it was rather obvious, given the content of above section. You think it's not ?

Reply:
I can think of one student for which it wasn't that obvious. It may juste be an exception. We can keep it as is and eventually change it later based on user feedback.

Reply:
I tried something. Let me know if it looks Ok to you.

Reply:
The explanation looks good to me, and I think I get why the example on the wing area looks like this [value not on the same line as the name and unit of the variable (again maybe "line" is not the right term for XML)], but isn't it too different from the other examples where name, value and unit are all on the same line ?

Reply:
Actually, "line" is fine, since it applies about the file layout. But in terms of XML, the term is irrelevant because line breaks do not matter, so there is strictly no difference between the examples.
I wanted the first example to clearly show the path breakdown, which is why I added these additional line breaks. I really hope that anyone disturbed by the layout discrepancies between samples will have a second look and find out they are equivalent.

Reply:
We mentionned it yesterday but we saifthat the users that were going to read the documentation were already a bit "fluent" in Python, so I guess it makes sense to assume that they would either know that line breaks don't matter in XML or have the hindsight to find out on their own.